app/(main)/(drawer)/(apply-now)/edit-application/[pageID].web.tsx

```
@@ -0,0 +1,579 @@
+    const applicationData = applicationDoc.data() as IApplications;
+
+    if (applicationData) {
+      setNote(applicationData.message || "");
+      setQuotation(applicationData.quotation || "");
+      setTimelineData(
+        applicationData.timeline ? new Date(applicationData.timeline) : null
+      );
+      setFileAttachments(applicationData.fileAttachments || []);
+      setAnswers(
+        applicationData.answersFromInfluencer.reduce(
+          (acc, curr) => ({ ...acc, [curr.question]: curr.answer }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)